build/common.props

```diff
@@ -4,8 +4,8 @@
     <TargetFramework>netstandard2.1</TargetFramework>
     <LangVersion>latest</LangVersion>
     <MajorVersion>4</MajorVersion>
-    <MinorVersion>2</MinorVersion>
```

Comment:
For a patch, this should be targeting the same minor with a different patch. Is there a reason this is jumping here?

Reply:
Tagging Jameson here. @VpOfEngineering

Reply:
Yes, it was a merge conflict (remote was 4.2.1, local was 4.3.1) and I wrongly selected 4.2.1 for this file. We could re-do the PR with that selection switched